### PR TITLE
Update estraverse to ^4.1.0

### PR DESCRIPTION
**Pull request by zckrs**

### CODE REVIEW

File: test/compare-acorn-es6/exports.expected.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
-export * from 'OK';
+export {
```
Comment (michaelficarra) [inline test/compare-acorn-es6/exports.expected.js]: This isn't a program.

**Comment by TrySound** [comment]
@michaelficarra Did you find the problem?

**Comment by michaelficarra** [comment]
@TrySound I wasn't planning on looking for the problem.

**Comment by TrySound** [comment]
@michaelficarra Decision of this problem:))
https://github.com/estools/escodegen/pull/245

**Comment by michaelficarra** [comment]
Ah, I understand now. @TrySound did you ever open issues for those?

**Pull request closed without merging**